corehq/form_processor/submission_post.py

```diff
@@ -445,6 +448,38 @@ def do_post_save_actions(case_db, xforms, case_stock_result):
             })
             raise PostSaveError
 
+    @staticmethod
+    def index_case_search(instance, case_models):
+        if not instance.metadata or instance.metadata.deviceID != FORMPLAYER_DEVICE_ID:
+            return
+
+        from corehq.apps.case_search.models import CaseSearchConfig, case_search_enabled_for_domain
+        if not case_search_enabled_for_domain(instance.domain):
+            return
+
+        config = CaseSearchConfig.objects.get_or_none(pk=instance.domain)
```

Comment:
is it worth creating a cached accessor for this?

Reply:
yeah, that sounds like a nice perf improvement - 9f407aedebd074d93e09cbab1882009d8982330b